dist/styles/api-console-light-theme.css

```diff
@@ -1232,15 +1232,22 @@ th {
   }
 }
 
-.raml-console-theme-toggle {
+.raml-console-meta-button-group {
   position: absolute;
   top: 0;
   right: 0;
-  display: inline-block;
+}
+
+.raml-console-meta-button-container {
+  position: relative;
+  float: right;
+}
+
+.raml-console-meta-button {
+  display: block;
   padding: 5px 8px;
-  border: solid rgba(0, 0, 0, 0.1);
+  border: 1px solid rgba(0, 0, 0, 0.1);
```

Comment:
Why setting the width and overwrite it below? You are doing the same in the other theme.